qa-tests-backend/src/main/groovy/objects/Notifiers.groovy

```diff
@@ -124,7 +124,13 @@ class SplunkNotifier extends Notifier {
 
     SplunkNotifier(boolean legacy, String collectorServiceName, int port, String integrationName = "Splunk Test") {
         splunkPort = port
-        notifier = NotifierService.getSplunkIntegrationConfig(legacy, collectorServiceName, integrationName)
+        String hecToken  = "00000000-0000-0000-0000-000000000000" // default for Splunk 6
+        if (!legacy) {
+            // For current Splunk images an ingest token needs to be created
+            log.info("Creating HEC ingest token")
+            hecToken = SplunkUtil.createHECToken(splunkPort)
+        }
```

Comment:
```suggestion
            hecToken = SplunkUtil.createHECToken(splunkPort)
        }
        log.info("Using HEC ingest token: ${hecToken}")
```

Reply:
Do we want to assert that `SplunkUtil.createHECToken(splunkPort)` returned something reasonable?

Reply:
Yes, I've added an assert to `createHECToken` to ensure the token has the expected length